bioblend/galaxy/invocations/__init__.py

```diff
@@ -247,8 +250,37 @@ def get_invocation_biocompute_object(self, invocation_id):
         url = self._make_url(invocation_id) + '/biocompute'
         return self._get(url=url)
 
+    def wait_for_invocation(self, invocation, timeout_seconds=10):
+        """
+        Wait for an invocation to be scheduled.
+
+        :type invocation: dict
+        :param invocation: Invocation to wait for.
+
+        :type timeout_seconds: int
+        :param timeout_seconds: Timeout in seconds. If the invocation is not scheduled
+                                after this timeout, an InvocationNotScheduledException
+                                is raised.
+        """
+        galaxy_version = os.environ.get('GALAXY_VERSION', None)
+        is_newer = galaxy_version == 'dev' or galaxy_version >= 'release_19.09'
```

Comment:
On reading this again, I think this is too complicated, `/api/workflows/.../invocations/...` might be deprecated in newer versions, but it still is totally functional. I would just rely on this so we don't have to check the Galaxy version.

(As an aside, although this way of checking the version might be passing CI tests, normal users don't have the GALAXY_VERSION env variable defined. So this does not work in practice, the correct way would be with `gi.config.get_version()`)